docs/docs/guides/developer_guides/getting_started.md

```diff
@@ -0,0 +1,252 @@
+Public key:      0x24d07ed6baff3e12dd76e7299a771bfa430eed9e25b155e3253c958b0a93551f2012b9d0238581717e27b296124272053f53757942972df98ad6602369a4d2c92bdd202abb07ab331dd1e3d5595f00be5a1c9d3c3244e07453f38e83417b2a381a0039f104c8f2214611b7a645deb01bda7c79835a46cf6c78cd485bd4d1372a10844f6c07f18a8435c5998d7739be448c6b5c15688b7c228b0a7dc24b038de700d0d4af57aa6b07c4471bcdcc5228a17b1ac03f2aae84c33dd2933b6b5a1c70179399adacbfc836c8bf24741ea78d404fc803fe09cfc570560b16c1169a40be1f20f153e8e0e2f9856ee526723b49c6da81e82b382a8770e31c78f25a1784cc
+Secret key:     0x1a9af8f75b3430223b628f7b423667da14304d376a82cfd31dfe138d00a20c61
+Partial address: 0x198022f6ec9b51f3af4cdc731adf606c0143ad35f7947ad604db50e9310a31ac
+Salt:            0x0000000000000000000000000000000000000000000000000000000000000000
+Init hash:       0x04630e3997204b1c417868aa449beb272c3c3bf0efe22315c5920665a6390db7
+Deployer:        0x0000000000000000000000000000000000000000000000000000000000000000
+
+Waiting for account contract deployment...
+Deploy tx hash:  09245ec0d905a461c717d6f70600e3c13b8746ee97785fd491b58f5f56111854
+Deploy tx fee:   200013616
+```
+
+You can double check by running `aztec get-accounts`.
+
+You will be needing that `Address` and `Secret key` fields later! To make it easier to reference, export them like this:
+
+```bash
+export ACCOUNT_ADDRESS=<ADDRESS>
+export SECRET_KEY=<SECRET_KEY>
+```
+Make sure to replace the `<ADDRESS>` and `<SECRET_KEY>` with the actual results that were printed.
+
+With this new account, let's get some local test tokens!
+
+## Deploying a contract
+
+The sandbox comes with some contracts that you can deploy and play with. One of these is an example token contract. 
+
+Deploy it with this:
+
+```bash
+aztec deploy TokenContractArtifact --secret-key $SECRET_KEY --args $ACCOUNT_ADDRESS test-token TST 18
+```
+
+This takes 
+- the contract artifact as the argument, which is `TokenContractArtifact`
+- the secret key of the deployer account, which we exported as `$SECRET_KEY`
+- the args that the contract constructor takes, which is the `admin` (`$ACCOUNT_ADDRESS`), `name` (`test-token`), `symbol` (`TST`), and `decimals` (`18`).
+
+On successful deployment, you should see something like this:
+
+```bash
+ aztec:cli [INFO] Using wallet with address 0x22d82b1ef045dfb4c3e0c4eac41fcf04ba29a3468d5ef5a123e4754064a4038d +0ms
+Contract deployed at 0x1eaa85d27c6ceed91f39a4885e2996f1f6de5d861c40dc9f0cfba43011b15243
+Contract partial address 0x19e957e72ec743a91c341b6e148e41f57c503741ea95e5b6e7158892dcb9f347
+Contract init hash 0x2f4bfd6cfa8f56f133bd0cd5ddfd1cb973436e0d385019491ce8fb950ae7eaf4
+Deployment tx hash: 12e43da89dcb4fbc693419d37e2a01f8f381da49b2cd1ae8492cc91b98aafd08
+Deployment salt: 0x115dd94b02568fc1f875fa782805151557684450f6aee058b28f9f010593e671
+Deployment fee: 200943060
+```
+
+Nice one! Let's export the `Contract deployed at` address so we can use it to call the contract:
+
+```bash
+export CONTRACT_ADDRESS=<CONTRACT_ADDRESS>
```

Comment:
Same here, the `aztec-wallet` is pretty useful, would could either pass `-a contracts:token_contract` and then use it like that, or just do nothing and it will auto alias the contract as `last` so you can pass just `last` or `contracts:last`